mllib/src/test/scala/org/apache/spark/ml/tree/impl/BaggedPointSuite.scala

```diff
@@ -91,8 +91,7 @@ class BaggedPointSuite extends SparkFunSuite with MLlibTestSparkContext  {
         baggedRDD.map(_.subsampleCounts.map(_.toDouble)).collect()
       EnsembleTestHelper.testRandomArrays(subsampleCounts, numSubsamples, expectedMean,
         expectedStddev, epsilon = 0.01)
-      // should ignore weight function for now
-      assert(baggedRDD.collect().forall(_.sampleWeight === 1.0))
+      assert(baggedRDD.collect().forall(_.sampleWeight === 2.0))
```

Comment:
just trying to understand, why did the sample weight change in this test?

Reply:
because this testsuite meet conditions: withReplacement=false, numSubsamples!=1,
it will call the modified `convertToBaggedRDDSamplingWithoutReplacement`,

and the `extractSampleWeight` here is `(_: LabeledPoint) => 2.0`, so output baggedPoints will have sampleWeight==2.0